src/vireshJJun20/Assignment23/ArrayIntersection.java

```diff
@@ -0,0 +1,86 @@
+/*
+Program 2: Find intersection(common elements) of two arrays.
+Input:    arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+
+Output: 10.45, 18.35
+Method Signature : double[] getIntersectionOfArray(double[] arr1, double[] arr2)
+*/
+
+
+package vireshJJun20.Assignment23;
+
+public class ArrayIntersection {
+
+	static int count;
+	boolean flag = false;
+
+	double[] editArray(double[] ar) { // takes input array and removes duplicates and then returns.
+		double[] temp = new double[ar.length];
+		for (int outerIndex = 0; outerIndex < ar.length; outerIndex++) {
+			for (int innerIndex = 0; innerIndex < ar.length; innerIndex++) {
```

Comment:
Loop is iterating for more number of time and line number 22 to 27 can be written in better way.

for (int outerIndex = 0; outerIndex < arr1.length; outerIndex++) {
			boolean flag = true;
			for (int innerIndex = 0; innerIndex < count; innerIndex++) {
				if (arr1[outerIndex] == tempAr[innerIndex]) {
					flag = false;
					break;
				}
			}
			if (flag == true) {
				tempAr[count] = arr1[outerIndex];
				count++;
			}
		}

I wrote it on the fly to give you an idea. please verify and revert back if you agree.

Reply:
Hi Krishna, I have observed your code snippet, and made changes in innerloop (innerIndex < count). if we skip the else condition (else {
					flag = true;
					break;
				}), 
then the Arr1 duplicate value remains as it is in a resultant array.  and the output reflects as below.
![image](https://user-images.githubusercontent.com/66830247/86515487-c91f3b00-be36-11ea-91ff-9ef7ad985888.png)

In case if the else condition is present, then output reflects correctly. As below:

![image](https://user-images.githubusercontent.com/66830247/86515517-0d124000-be37-11ea-9b11-da748645016f.png)

Reply:
And lastly if I change the if ()condition, then the output is blank. as highlighted below:

![image](https://user-images.githubusercontent.com/66830247/86515711-9d04b980-be38-11ea-9ea3-dc491ed56983.png)